docs/source/getting-started/quickstart.rst

```diff
@@ -97,9 +97,19 @@ Let's apply Laplace noise to a value.
             :end-before: /demo
 
 This is obviously not the easiest way to add noise to a number,
-but it demonstrates a number of OpenDP patterns:
+but it demonstrates a number of low-level OpenDP patterns:
```

Comment:
I might drop "This is obviously not the easiest way to add noise to a number," unless you have an affinity to it.

Reply:
will do!

Reply:
Now:
> This demonstrates a number of low-level OpenDP patterns: